yetanotherco/aligned_layer
# fix: risc0 verifier public inputs in journal and public inputs may not match

> [!WARNING]
> This PR will make the operators unable to verify the current risc0 proof, they will need to update.

# Description

- This PR separates the risc0 proof from the public inputs.

# To Test

- Run `make build_all_ffi_macos` or `make build_all_ffi_linux` depending on your OS.
- Run `make test_risc_zero_rust_ffi`.
- Run `make test_risc_zero_go_bindings_macos`.
- Start everything as usual and send `make batcher_send_risc0_task`.
- Make sure everything is working correctly
